lib/facebook/messenger/incoming.rb

```diff
@@ -25,7 +26,8 @@ module Incoming
         'referral' => Referral,
         'message_echo' => MessageEcho,
         'message_request' => MessageRequest,
-        'payment' => Payment
+        'payment' => Payment,
+        'policy-enforcement' => PolicyEnforcement
```

Comment:
Let's underscore case this to keep it consistent with `message_echo` and `message_request`.